packages/nvm.rb

```diff
@@ -16,15 +16,10 @@ def self.install
     system "rm -rf #{CREW_DEST_PREFIX}/share/nvm/test"
   end
 
-  def self.postinstall
+  def self.remove
     puts
-    puts "To complete the installation, execute:".lightblue
-    puts "source ~/.bashrc".lightblue
-    puts
-    puts "To uninstall, execute the following:".lightblue
-    puts "crew remove nvm".lightblue
+    puts "Don't forget to run:"
     puts "rm -rf #{CREW_PREFIX}/share/nvm".lightblue
```

Comment:
Why not prompt to remove?

Reply:
For some reason I thought it was better at the time. Would you like that reverted?

Reply:
Is it difficult to prompt?  This is what I was thinking:
```
    if Dir.exists? "#{CREW_PREFIX}/share/nvm"                                                                                                                                               
      puts                                                                                                                                                                                    
      print "Would you like to remove #{CREW_PREFIX}/share/nvm? [y/N] "                                                                                                                     
      response = STDIN.getc                                                                                                                                                                   
      case response                                                                                                                                                                           
      when "y", "Y"                                                                                                                                                                           
        FileUtils.rm_rf "#{CREW_PREFIX}/share/nvm"                                                                                                                                          
        puts "#{CREW_PREFIX}/share/nvm removed.".lightred                                                                                                                                   
      else                                                                                                                                                                                    
        puts "#{CREW_PREFIX}/share/nvm saved.".lightgreen                                                                                                                                   
      end                                                                                                                                                                                     
      puts                                                                                                                                                                                    
    end
```

Reply:
I'm going to work on a function to do that, I think we can leave this as is for the time being.